packages/core/src/lib/TwaGenerator.ts

```diff
@@ -250,16 +250,23 @@ export class TwaGenerator {
     }));
   }
 
-  private async writeWebManifest(webManifestUrl: URL, targetDirectory: string): Promise<void> {
-    const response = await fetch(webManifestUrl);
+  private async writeWebManifest(twaManifest: TwaManifest, targetDirectory: string): Promise<void> {
+    if (!twaManifest.webManifestUrl) {
```

Comment:
Are we just failing silently in this case?

Reply:
Hum, in fact, this is being checked just before calling the method. Since this is an internal method, I can probably just remove the check.

Reply:
Added an error instead of just removing the check, as the linter will be unhappy if just removing it.